package.json

```diff
@@ -19,7 +19,7 @@
   "author": "Mikael Brevik",
   "license": "MIT",
   "peerDependencies": {
-    "marked": "^1.0.0 || ^2.0.0"
+    "marked": "^1.0.0 || ^2.0.0 || ^3.0.0"
```

Comment:
I thought the `||` syntax was invalid, but I found this site https://semver.npmjs.com/ .

In another language I would use `>=1 <4`

Reply:
One tiny difference is with this syntax you can use `^1.1 || ^2.1` (i.e. require `>= 1.1` OR `>= 2.1`) whereas `>= 1 < 4` would not disallow `2.0`. I don't have a strong opinion on which is better though.